functions.p8

```diff
@@ -1,16 +1,15 @@
-pico-8 cartridge // http://www.pico-8.com
-version 18
-__lua__
--- pico8lib functions library
--- by sparr
+--- @module functions
+--- Manipulating and creating functions
+-- @author sparr
```

Comment:
If we're ditching the pico8 cart header and compatibility, I don't feel the need to keep the author line. You can leave yours if you want, but I'm fine removing mine. With luck eventually most files will have multiple contributors anyway. I already moved acknowledgements for some of the third party code out to its own comments.

Reply:
Sweet. That will simplify things a little bit.